07_perfect_forwarding/07_02_forwarding_reference/07_02_00_type_deduction.cpp

```diff
@@ -4,7 +4,7 @@
 #include <iostream>
 
 template <typename T>
-void foo(T) {
+void foo(T& t) {
```

Comment:
```suggestion
void foo(T&) {
```

Reply:
Here we want to talk about type of `t`.

Reply:
Yep. `t` is fundamental here because we would like to explain `t` type rather then `typename T` deduction